src/dashboard/Dashboard.tsx

```diff
@@ -1,41 +1,112 @@
+};
+
+const getMonthsBeforeNow = function (months: number): Date {
+  const d = new Date();
+  d.setMonth(d.getMonth() - months);
+  return d;
+};
+const oneMonthBeforeNow = getMonthsBeforeNow(1);
+const twoMonthsBeforeNow = getMonthsBeforeNow(2);
 
 export const Dashboard = () => {
+  const monthlyOrders = useGetList("commands", {
+    filter: { date_gte: oneMonthBeforeNow.toISOString() },
+  });
+  const pendingOrders = useGetList("commands", {
+    filter: { status: "ordered" },
+    pagination: { page: 1, perPage: 25 },
+  });
+  const pendingReviews = useGetList("reviews", {
+    filter: { status: "pending" },
+    pagination: { page: 1, perPage: 25 },
+  });
+  const newCustomers = useGetList<Customer>("customers", {
+    filter: {
+      first_seen_gte: twoMonthsBeforeNow.toISOString(),
+      has_ordered: true,
+    },
+    pagination: { page: 1, perPage: 25 },
+  });
+
   return (
     <Container sx={{ padding: 2 }}>
       <Box display="grid" gridTemplateColumns="repeat(4, 1fr)" gap={2}>
         <Box gridColumn="span 1">
-          <Card component={Paper}>
-            <CardContent>Monthly revenue</CardContent>
+          <Card component={Paper} sx={cardSx}>
+            <CardContent>
+              <DashboardCardHeader
+                icon={<AttachMoneyIcon fontSize="large" color="secondary" />}
+                title="Monthly Revenue"
+                content="2 481 $US"
+              />
+            </CardContent>
           </Card>
         </Box>
 
         <Box gridColumn="span 1">
-          <Card component={Paper}>
-            <CardContent>New Orders</CardContent>
+          <Card component={Paper} sx={cardSx}>
+            <CardContent>
+              <DashboardCardHeader
+                icon={<ShoppingCartIcon fontSize="large" color="secondary" />}
+                title="New Orders"
+                content="17"
+              />
+            </CardContent>
           </Card>
         </Box>
 
         <Box gridColumn="span 1" gridRow="span 3">
-          <Card component={Paper} sx={{ height: "100%" }}>
+          <Card component={Paper} sx={{ height: "100%", ...cardSx }}>
             <CardContent>Pending Reviews</CardContent>
           </Card>
         </Box>
 
         <Box gridColumn="span 1" gridRow="span 3">
-          <Card component={Paper} sx={{ height: "100%" }}>
-            <CardContent>New Customers</CardContent>
+          <Card component={Paper} sx={{ height: "100%", ...cardSx }}>
+            <CardContent>
+              <DashboardCardHeader
+                icon={<PersonAddIcon fontSize="large" color="secondary" />}
+                title="New Customers"
+                content={String(newCustomers.total)}
+              />
+              <Divider />
+              {newCustomers.data &&
+                newCustomers.data.map((customer) => (
+                  <Box sx={{ margin: 2 }}>
+                    <CustomerField key={customer.id} record={customer} />
+                  </Box>
```

Comment:
To avoid passing the `record`, you could also wrap this inside a `RecordContextProvider`. Not very important though as all react-admin will get the record from either the props or the context. The one from the prop takes precedence

Reply:
Eventually I replaced this with a `SimpleList` in [this commit](https://github.com/slax57/admin-e-commerce-demo/pull/12/commits/2faf973e22c9fc7fc6b97a5c6c2abab5d2cee892), removing the need to pass the record manually 🙂